### PR TITLE
De37909/fix pending copy

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

File: src/components/cpd-admin.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ class CpdAdmin extends BaseMixin(LitElement) {
 	}
 	render() {
 		if (this.pageData && this.pageData.page === 'cpd-manage-targets') {
-			return html`<d2l-cpd-manage-targets jobTitle="${this.pageData.jobTitle}"></d2l-cpd-manage-targets>`;
+			return html`<d2l-cpd-manage-targets .jobTitle="${this.pageData.jobTitle}"></d2l-cpd-manage-targets>`;
```
Comment (jwong-d2l) [inline src/components/cpd-admin.js]: DE37920

File: src/components/cpd-my-records.js [hunk 2]
```diff
@@ -381,7 +381,7 @@ class MyCpdRecords extends BaseMixin(LitElement) {
 						</h2>
 						<div class="printLink">
 							<d2l-icon icon="tier1:print"></d2l-icon>
-							<d2l-link target="_blank" href="${CpdRoutes.Report(this.viewUserId)}">${this.localize('printRecords')}</d2l-link>
+							<d2l-link target="_blank" href="${CpdRoutes.UserReport(this.viewUserId)}">${this.localize('printRecords')}</d2l-link>
```
Comment (jwong-d2l) [inline src/components/cpd-my-records.js]: DE37921

File: src/components/cpd.js [hunk 3]
```diff
@@ -59,9 +59,10 @@ class Cpd extends BaseMixin(LitElement) {
 			return html`<d2l-cpd-manage-targets></d2l-cpd-manage-targets>`;
 		}
 		if (this.pageData.page === 'cpd-add-record' || this.pageData.page === 'cpd-edit-record') {
+
 			if (this.pageData.awardData) {
 				return html`
-				<d2l-cpd-add-record awardRecord="${this.pageData.awardData}"></d2l-cpd-add-record>
+				<d2l-cpd-add-record .awardRecord="${JSON.parse(this.pageData.awardData)}"></d2l-cpd-add-record>
```
Comment (jwong-d2l) [inline src/components/cpd.js]: DE37909

File: src/components/cpd-manage-targets.js [hunk 4]
```diff
@@ -154,7 +154,7 @@ class ManageCpdTargets extends BaseMixin(LitElement) {
 							<div id="structured">
 								<label for="structuredHours">${this.localize('hours')}</label>
 								<d2l-input-text
-									@change="${this.onDialogDataUpdated}"
+									@blur="${this.onDialogDataUpdated}"
```
Comment (jwong-d2l) [inline src/components/cpd-manage-targets.js]: DE37910

File: src/components/cpd-my-records.js (hunk 2 above)
Comment (TomMichielsD2L) [reply src/components/cpd-my-records.js]: We also need to fix line 149 on cpd-record-report to ask for the user's progress. Which also means that we need to add a route for this in cpdRoutes, and we need to update the function in the cpd-service

Comment (jwong-d2l) [reply src/components/cpd-my-records.js]: Not sure what you mean.... can you either push changes, or I can talk to you when I get in?

File: src/services/cpd-service.js [hunk 5]
```diff
@@ -103,8 +103,11 @@ export class CpdService {
 		return this.getRequest(api_path);
 	}
 
-	static getProgress() {
-		return this.getRequest(CpdRoutes.Path(CpdRoutes.Progress));
+	static getProgress(userId) {
+		if (!userId) {
+			return this.getRequest(CpdRoutes.Path(CpdRoutes.Progress));
+		}
+		return this.getRequest(CpdRoutes.Path(CpdRoutes.UserProgress(userId)));
```
Comment (TomMichielsD2L) [inline src/services/cpd-service.js]:
```suggestion
		return this.getRequest(CpdRoutes.Path(CpdRoutes.UserProgress(userId)));
```

File: src/helpers/cpdRoutes.js [hunk 6]
```diff
@@ -13,11 +13,13 @@ export class CpdRoutes {
 	static get Question() { return 'question'; }
 	static QuestionId(questionId) { return `question/${questionId}`; }
 	static get Record() { return 'record'; }
+	static RelativePath(action) {return `/d2l/api/customization/cpd/1.0/${action}`;}
```
Comment (BraydonKains) [inline src/helpers/cpdRoutes.js]: What's the difference between `RelativePath` and `Path`? Kind of looks like the same thing, and we seem to use both.

Comment (TomMichielsD2L) [reply src/helpers/cpdRoutes.js]: I did not notice path. I will fix it

**Pull request merged**